packages/render-fidelity-tools/test/renderers/model-viewer/index.html

```diff
@@ -60,7 +60,9 @@
         // the poster has completely faded away. 
         requestAnimationFrame(()=>{
           requestAnimationFrame(()=>{
-            this.dispatchEvent(new CustomEvent('model-ready'));
+            requestAnimationFrame(()=>{
+              this.dispatchEvent(new CustomEvent('model-ready'));
```

Comment:
Might be nice to add a comment about how it seems like three rafs are needed to get the poster to fade out reliably. Nice to document work-arounds.